### PR TITLE
Docs rework

### PR DESCRIPTION
**Pull request by joypoddar**
Updated `docs/intro.md` with improvements wrt issue #10 

Linking redundant issue #29 as recommended by @atapas.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-req.yml
Comment (Sachin-chaurasiya) [inline .github/ISSUE_TEMPLATE/feature-req.yml:1]: @joypoddar , you have separate PR for this changes here #31 .
only keep changes related to this PR.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature-req.yml
Comment (joypoddar) [reply .github/ISSUE_TEMPLATE/feature-req.yml:1]: @Sachin-chaurasiya I am new to contributing to OSS. I did create a separate PR, but somehow it pulled both the commits in this PR. I wanted the `docs: rework (6f0a27c)` commit here. I am not sure how to do that...

I appreciate your patience and understanding.

Comment (Sachin-chaurasiya) [reply .github/ISSUE_TEMPLATE/feature-req.yml:1]: Hello @Sachin-chaurasiya , No worries, we learn by doing mistakes.

Here are the steps
- `git checkout main`
-  `git remote add upstream https://github.com/reactplay/docs.git`
-  `git fetch upstream`
-  `git merge upstream/main`
-  `git push origin main`

Once these steps are done, you can merge main branch into this branch

`git checkout doc-rework`
`git merge main`

Thanks.

Comment (joypoddar) [reply .github/ISSUE_TEMPLATE/feature-req.yml:1]: @Sachin-chaurasiya  I have done the above steps. 

How do I resolve the changes you requested before the below comment? Do I leave it as it is as suggested by @atapas in discord or is there something else that I have to do?
> @joypoddar , you have separate PR for this changes here #31 . only keep changes related to this PR.

Comment (Sachin-chaurasiya) [reply .github/ISSUE_TEMPLATE/feature-req.yml:1]: No worries @joypoddar, It's resolved.

**Comment by joypoddar** [comment]
@Sachin-chaurasiya I am unable to view the changes requested

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya I am unable to view the changes requested

It's resolved @joypoddar

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]
Thanks for the PR @joypoddar

**Review by atapas — APPROVED** [review]

**Pull request merged**